lib/sqlalchemy/dialects/oracle/cx_oracle.py

```diff
@@ -61,8 +61,7 @@
     engine = create_engine("oracle+cx_oracle://scott:tiger@myalias/?encoding=UTF-8&nencoding=UTF-8")
 
 Users of Oracle Cloud should use this syntax and also configure the cloud
-wallet as shown in cx_Oracle documentation `Connecting to Autononmous Databases
-<https://cx-oracle.readthedocs.io/en/latest/user_guide/connection_handling.html#autonomousdb>`_.
+wallet as shown in cx_Oracle documentation :external+cx-oracle:ref:`autonomousdb`.
```

Comment:
in your PR at #11762 , intersphinx doesnt help much unless we are trying to guard against the page itself changing.   the link that's there was probably taken from the URL bar and we simply didnt know the link had a dedicated anchor.